build.gradle.kts

```diff
@@ -107,3 +110,23 @@ val hivemqEdgeZip by tasks.registering(Zip::class) {
     }
 }
 
+
+
+val edgeProjectsToUpdate = setOf(
+    "hivemq-edge",
+    "hivemq-edge-module-http",
```

Comment:
possible improvement: can we determine the list of modules dynamically from checking the modules folder ? This would remove a potential source for mistakes.

Reply:
hm i guess we could derive it from the dependencies as we have an own configuration for the modules, but unfortunately I dont know how this works as the configurations are not builds